### ISSUE TITLE
Compatible with SP3, server variables?

### ISSUE THREAD

**Issue by claykimber** [Brown-University-Library/django-shibboleth-remoteuser#92]
First of all, thank you for your work on this project.  Much appreciated.

Is this project compatible with Shibboleth Service Provider version 3 (SP3)?  Also, does the project make use of server variables as outlined in the [SP3 Attribute Access documentation](https://wiki.shibboleth.net/confluence/display/SP3/AttributeAccess)?

**Comment by bcail** [comment]
Yup, it's compatible with SP3, and you can use server variables (as opposed to custom headers). Either way, the info shows up in the django request and you can use it with this project.

**Comment by claykimber** [comment]
Excellent, thank you!
